src/main/java/com/google/sps/data/Destination.java

```diff
@@ -48,6 +48,35 @@
 
   private Destination() {}
 
+  // Getter methods
+  public String getName() {
+    return this.name;
+  }
+
+  public Obscurity getDifficulty() {
+    return this.level;
+  }
+
+  public String getCity() {
+    return this.city;
+  }
+
+  public static Obscurity stringToEnum(String difficulty) {
```

Comment:
In the future if you want to add another static method that returns an enum, this name will be confusing.  Suggest renaming to toObscurity or something similar.